staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -31,6 +31,9 @@ import (
 // StacktracePred returns true if a stacktrace should be logged for this status.
 type StacktracePred func(httpStatus int) (logStacktrace bool)
 
+// handler func after execute server http
+type AfterServeHTTPHandler func(w http.ResponseWriter, r *http.Request)
```

Comment:
This looks a bit too general to be added to the http logger (it could just be a generic utility, e.g. `httputil.AddResponseHook(...)`)

Reply:
updated, plz take a look :)